tests/foreman/api/test_subnet.py

```diff
@@ -354,11 +355,26 @@ def test_positive_update_subnet_parameter_host_impact():
 
         1. The inherited subnet parameter in host should have
             updated name and value
-        2. The inherited subnet parameter in host enc should have
-            updated name and value
 
     :BZ: 1470014
     """
+    parameter = [{'name': gen_string('alpha'), 'value': gen_string('alpha')}]
+    org = target_sat.api.Organization().create()
+    loc = target_sat.api.Location(organization=[org]).create()
+    org_subnet = target_sat.api.Subnet(
+        location=[loc], organization=[org], subnet_parameters_attributes=parameter
+    ).create()
```

Comment:
you can have assertions after the first subnet creation to verify if the parameter name and value were set correctly

Reply:
thanks for your suggestion. assertion point is added.